src/ducks/apps/components/ApplicationRouting/InstallRoute.jsx

```diff
@@ -27,18 +29,8 @@ export const InstallRoute = ({ getApp, isFetching, parent, redirectTo }) => {
   const redirectToApp = () => redirectTo(appPath)
 
   const redirectToConfigure = () => {
-    const redirectionPath = searchParams.get('redirectAfterInstall')
-
-    if (redirectionPath) {
-      const subDomainType = client.getInstanceOptions().subdomain
-      const { slug } = deconstructCozyWebLinkWithSlug(
-        redirectionPath,
-        subDomainType
-      )
-      return openApp(webviewIntent, {
-        slug,
-        related: redirectionPath
-      })
```

Comment:
Are we sure that no other app use this redirectAfterInstall?

Reply:
Oui, certain.

Reply:
How can you be sure? Did you check the code on our partners apps? 

I'm pretty sure the code is not used, but removing this kind of functionality is not so simple. This is just a simple warning ;)